style.css

```diff
@@ -0,0 +1,133 @@
+@font-face {
+    font-family: 'NeueMontrealBook';
+    src: url('PPNeueMontreal-Book.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'NeueMontrealItalic';
+    src: url('PPNeueMontreal-Italic.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'NeueMontrealMedium';
+    src: url('PPNeueMontreal-Medium.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'NeueMontrealSemiBoldItalic';
+    src: url('PPNeueMontreal-SemiBolditalic.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'NeueMontrealThin';
+    src: url('PPNeueMontreal-Thin.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'GoshaSansBold';
+    src: url('PPGoshaSans-Bold.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'GoshaSansRegular';
+    src: url('PPGoshaSans-Regular.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+.top_container {
+    display: flex;
+    justify-content: center;
+    align-content: center;
+    position: relative;
+}
+
+header {
+    font-family: GoshaSansBold, sans-serif;
+    display: flex;
+    justify-content: center;
+    align-content: center;
+    background-color: #661D09;
+    color: #FFF;
+}
+
+.app_title {
+    font-family: GoshaSansBold, sans-serif;
```

Comment:
You should be able to just set this font in the body selector to use it everywhere, instead of adding it in all these different places